mmdet/core/evaluation/eval_hooks.py

```diff
@@ -70,16 +123,38 @@ def evaluation_flag(self, runner):
     def after_train_epoch(self, runner):
         if not self.evaluation_flag(runner):
             return
+        current_ckpt_path = osp.join(runner.work_dir,
+                                     f'epoch_{runner.epoch + 1}.pth')
+        json_path = osp.join(runner.work_dir, 'best.json')
```

Comment:
Is it better to save the best info into the pth file instead of creating another json?

Reply:
The checkpoint hook is sperate from eval hook. 
We may create another PR in MMCV to implement it.